### PR TITLE
v7.11.1

### PR DESCRIPTION
**Pull request by MBilalShafi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,88 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.11.1
+
+_Jul 25, 2024_
+
+We'd like to offer a big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Add Icelandic (is-IS) and Norwegian Nynorsk (nn-NO) locales on the Data Grid
```
Comment (MBilalShafi) [inline CHANGELOG.md:12]: Let me know if there's something else that should be highlighted.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,88 @@
...
+- 🌍 Improve Norwegian bokmål (nb-NO) and German (de-DE) locales on the Data Grid
```
Comment (flaviendelangle) [inline CHANGELOG.md:13]: Nitpickest nitpick :laughing: 

```suggestion
- 🌍 Improve Norwegian Bokmål (nb-NO) and German (de-DE) locales on the Data Grid
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,88 @@
...
+- 🌍 Improve Norwegian Nynorsk (nn-NO) locale on the Date and Time Pickers
```
Comment (LukasTy) [inline CHANGELOG.md:14]:
```suggestion
- 🌍 Add Norwegian Nynorsk (nn-NO) locale on the Date and Time Pickers
```

File: CHANGELOG.md (hunk 1 above)
Comment (LukasTy) [reply CHANGELOG.md:12]: Maybe "Allow `Zoom` to be controllable"? 🤔 
cc @mui/xcharts

**Review by LukasTy — APPROVED** [review]

**Review by noraleonte — APPROVED** [review]
LGTM 🚀

**Review by arthurbalduini — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,88 @@
...
+- [l10n] Add Norwegian nynorsk (nn-NO) locale and improve Norwegian bokmål (nb-NO) locale (#13588) @AnderzL7
+- [l10n] Improve German (de-DE) locale (#13910) @lhilgert9
+
+#### `@mui/x-data-grid-pro@7.11.1` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@7.11.1`.
+
+#### `@mui/x-data-grid-premium@7.11.1` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@7.11.1`.
```
Comment (cherniavskii) [inline CHANGELOG.md:38]:
```suggestion
- [DataGrid] Do not miss to escape formulas in CSV export (#13888) @arminmeh
- [l10n] Add Icelandic (is-IS) locale (#13283) @magnimarels
- [l10n] Add Norwegian nynorsk (nn-NO) locale and improve Norwegian bokmål (nb-NO) locale (#13588) @AnderzL7
- [l10n] Improve German (de-DE) locale (#13910) @lhilgert9

#### `@mui/x-data-grid-pro@7.11.1` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')

Same changes as in `@mui/x-data-grid@7.11.1`.

#### `@mui/x-data-grid-premium@7.11.1` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')

Same changes as in `@mui/x-data-grid-pro@7.11.1`, plus:

- [DataGridPremium] Pass the `api` object to events (#13893) @pcorpet
- [DataGridPremium] Fix paste to selected cells (#13967) @romgrk
```

**Review by cherniavskii — APPROVED** [review]

**Pull request merged**